easybuild/easyconfigs/j/JasPer/JasPer-2.0.24-GCCcore-10.2.0.eb

```
@@ -0,0 +1,34 @@
+easyblock = 'CMakeMake'
+
+name = 'JasPer'
+version = '2.0.24'
+
+homepage = 'https://www.ece.uvic.ca/~frodo/jasper/'
+
+description = """
+ The JasPer Project is an open-source initiative to provide a free
+ software-based reference implementation of the codec specified in
+ the JPEG-2000 Part-1 standard.
+"""
+
+toolchain = {'name': 'GCCcore', 'version': '10.2.0'}
+toolchainopts = {'pic': True}
+
+github_account = 'jasper-software'
+source_urls = [GITHUB_SOURCE]
+sources = ['version-%(version)s.tar.gz']
+checksums = ['d2d28e115968d38499163cf8086179503668ce0d71b90dd33855b3de96a1ca1d']
+
+builddependencies = [
+    ('binutils', '2.35'),
+    ('CMake', '3.18.4'),
+]
+
+configopts = '-DJAS_ENABLE_DOC=OFF '
+
+sanity_check_paths = {
+    'files': ['bin/jasper', ('lib/libjasper.%s' % SHLIB_EXT, 'lib64/libjasper.%s' % SHLIB_EXT)],
+    'dirs': ['include'],
+}
+
```

@lexming maybe add a sanity_check_commands (--help exits with non-zero)?
```suggestion
sanity_check_commands = ['jasper --version']
```

---

sure :+1: